main.go

```diff
@@ -82,6 +82,12 @@ func list(c config.Config) {
 }
 
 func details(c config.Config, name string) {
+	_, err := c.Get(name)
+	if err != nil {
+		fmt.Println("It's not posible to show "+name+" as it doesn't exist.")
```

Comment:
I would change the text by `Alias not found` instead.

Reply:
Great, I've just done it and push it.